drivers/bus/pcix/CMakeLists.txt

```
@@ -48,6 +48,11 @@ add_library(pcix MODULE
 
 set_module_type(pcix kernelmodedriver)
 target_link_libraries(pcix arbiter)
+if(MSVC)
+    target_link_options(pcix PRIVATE /SECTION:.rsrc,!D)
+else()
+    # Not implemented
```

```suggestion
    add_custom_command(
        TARGET pcix POST_BUILD
        COMMAND objcopy --set-section-flags .rsrc=readonly $<TARGET_FILE:pcix>
        VERBATIM)
```
(Note: if "objcopy" doesn't work, try using `${CMAKE_OBJCOPY}` instead.)